### PR TITLE
Added Shooting Game-Enemy

### PR DESCRIPTION
**Pull request by zhairanicolepacheco**

### CODE REVIEW

**Review by RoCS2024 — CHANGES_REQUESTED** [review]
Remove unnecessary files

File: 04 Abstraction Exercises Part 1/Enemy-Application/src/main/java/com/shooting/game/enemy/Enemy.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package com.shooting.game.enemy;
+
+public class Enemy {
+    private int attackPoints;
+    private int hitPoints;
```
Comment (RoCS2024) [inline 04 Abstraction Exercises Part 1/Enemy-Application/src/main/java/com/shooting/game/enemy/Enemy.java:5]: make all instances of the enemy have the same attackpoints and hitpoints. meaning everytime an enemy instance spawn, it has the same AP and HP

**Review by RoCS2024 — CHANGES_REQUESTED** [review]
see comment

File: 04 Abstraction Exercises Part 1/Enemy-Application/src/main/java/com/shooting/game/enemy/Enemy.java (hunk 1 above)
Comment (zhairanicolepacheco) [reply 04 Abstraction Exercises Part 1/Enemy-Application/src/main/java/com/shooting/game/enemy/Enemy.java:5]: Done na po sir, though not sure if tama po ginawa

**Review by RoCS2024 — APPROVED** [review]
all good

**Pull request merged**